packages/app-earn-ui/src/components/molecules/InfoBox/InfoBox.tsx

```diff
@@ -0,0 +1,75 @@
+'use client'
+
+import { type ReactNode } from 'react'
+import clsx from 'clsx'
+
+import { Icon } from '@/components/atoms/Icon/Icon'
+import { Text } from '@/components/atoms/Text/Text'
+
+import styles from './InfoBox.module.scss'
+
+interface InfoBoxRow {
+  label?: string | ReactNode
+  value?: ReactNode | string | number
+  type: 'entry' | 'separator'
+}
+
+interface InfoBoxProps {
+  title?: string
+  rows?: InfoBoxRow[]
+  children?: ReactNode
+  className?: string
+  error?: string
+}
+
+export const InfoBox = ({ title, rows, children, className, error }: InfoBoxProps) => {
```

Comment:
I know that we have components typed like this one, but in general we are refactoring everything to FC<InfoBoxProps> approach

Reply:
Understood